livekit-ffi/protocol/room.proto

```diff
@@ -276,15 +288,16 @@ message RoomEvent {
     RoomSidChanged room_sid_changed = 15;
     ParticipantMetadataChanged participant_metadata_changed = 16;
     ParticipantNameChanged participant_name_changed = 17;
-    ConnectionQualityChanged connection_quality_changed = 18;
```

Comment:
For FFI we don't need to maintain message number consistency

Reply:
This is true, tho I'm thinking about changing the behaviour in the future. Safe to ignore for now